chia/_tests/core/full_node/test_full_node.py

```diff
@@ -3294,3 +3294,65 @@ def compare_unfinished_blocks(block1: UnfinishedBlock, block2: UnfinishedBlock)
     # Final assertion to check the entire block
     assert block1 == block2, "The entire block objects are not identical"
     return True
+
+
+@pytest.mark.anyio
+@pytest.mark.parametrize(
+    "condition_and_error",
+    [
+        (ConditionOpcode.ASSERT_HEIGHT_RELATIVE, "ASSERT_HEIGHT_RELATIVE_FAILED"),
+        (ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE, "ASSERT_HEIGHT_ABSOLUTE_FAILED"),
+    ],
+)
+async def test_peak_post_processing_added_tx(
+    condition_and_error: tuple[ConditionOpcode, str],
+    blockchain_constants: ConsensusConstants,
+    caplog: pytest.LogCaptureFixture,
+) -> None:
+    """
+    Tests peak post processing with regards to pending transactions due of
+    time-lock to make sure they get retried once the time-lock allows them to
+    be reconsidered.
```

Comment:
I think this comment should more directly say what *feature* is being tested and maybe which *cases* are covered. It's the `PendingTXCache`, right?